components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/IdentityCoreServiceComponent.java

```diff
@@ -171,6 +186,8 @@ protected void activate(ComponentContext ctxt) {
                     new IdentityCoreInitializedEventImpl(), null);
 
 
+        } catch (MigrationClientException e) {
+            throw e;
```

Comment:
Better to explain reason to throw. i.e. prevent the component gets activated. the OSGI thing.

Reply:
Added a comment